src/routes/committee/suggestions/+page.svelte

```diff
@@ -50,63 +83,73 @@
 			</StructuredListRow>
 		</StructuredListHead>
 		{#each data.suggestions as { song, user }, i}
-			<StructuredListRow>
-				<StructuredListCell>
-					{user}
-				</StructuredListCell>
-				<StructuredListCell>
-					{song.name}
-				</StructuredListCell>
-				<StructuredListCell>
-					{song.artist}
-				</StructuredListCell>
-				<StructuredListCell>
-					{song.genre}
-				</StructuredListCell>
-				<StructuredListCell>
-					{@const validUrl = isValidUrl(song.link)}
-					<Button
-						href={validUrl ? song.link : undefined}
-						kind="ghost"
-						size="small"
-						iconDescription={validUrl ? song.link : 'Invalid URL'}
-						icon={PlayFilledAlt}
-						disabled={!validUrl}
-					/>
-				</StructuredListCell>
-				<StructuredListCell>
-					<Button
-						kind="danger-tertiary"
-						size="small"
-						iconDescription="Like"
-						icon={song.user.id === 'KcRkWMQUEClLEeiccSD5' ? Bat : Favorite}
-					/>
-				</StructuredListCell>
-				<StructuredListCell>
-					<Button
-						kind="danger-tertiary"
-						size="small"
-						iconDescription="Delete"
-						icon={MusicRemove}
-						on:click={() => {
-							selectedSong = i
-							openDel = true
-						}}
-					/>
-					{#if song.remark && song.remark.length > 0}
+			{#if (filterFavourites && data.favouriteSongs[song.id]) || !filterFavourites}
```

Comment:
Misschien is het nicer om de filtering in de script te doen, heb je minder nesting